packages/grafana-ui/src/components/Select/types.ts

```diff
@@ -68,6 +69,8 @@ export interface SelectCommonProps<T> {
   onCreateOption?: (value: string) => void;
   onInputChange?: (value: string, actionMeta: InputActionMeta) => void;
   onKeyDown?: (event: React.KeyboardEvent) => void;
+  onMenuScrollToBottom?: (event: WheelEvent | TouchEvent) => void;
+  onMenuScrollToTop?: (event: WheelEvent | TouchEvent) => void;
```

Comment:
I know a lot of props aren't documented for this component, but could you maybe add some docstrings to these props?